src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
pathfinder+sender are designed for the case where the destination amount is known, which means the quoting must be done destination-to-source the find the source amount.

If the source amount where fixed instead, it would be the other way around.